Gemfile

```diff
@@ -39,6 +39,10 @@ gem "annotate"
 gem 'bootstrap', '>= 4.0.0.alpha3', '<= 4.1'
 gem "font-awesome-rails"
 
+# Slackbot framework https://github.com/slack-ruby/slack-ruby-bot
```

Comment:
I'm a little confused — how does `slack-ruby-bot` interact with the installed `slack-ruby-client`?